.env-example

```diff
@@ -34,6 +37,9 @@ AI_CONFIG_PREFIX=!config
 PROMPT_MODERATION_ENABLED = true
 PROMPT_MODERATION_BLACKLISTED_CATEGORIES = ["hate", "hate/threatening", "self-harm", "sexual", "sexual/minors", "violence", "violence/graphic"]
 
+# Access control, only allow whatsapp-chatgpt to react to specific phone numbers, comma-separated
+WHITELISTED_PHONE_NUMBERS=85233332222@c.us,85299996666@c.us,85288882222-12345678@g.us
```

Comment:
Remove the numbers from examples, put them in #

Reply:
the same, when I put the number in the env everything works, but when a number that is not in the whitelist send a command the error occurs

Reply:
You should not put numbers into the example, leave it empty and people have to put their own numbers in there.

Also make a boolean to enable/disable the whitelisting